cli/internal/app/install_claude_code.go

```diff
@@ -0,0 +1,181 @@
+package app
+
+import (
+	"fmt"
+	"log/slog"
+	"net/url"
+
+	"github.com/speakeasy-api/gram/cli/internal/app/logging"
+	"github.com/speakeasy-api/gram/cli/internal/claudecode"
+	"github.com/speakeasy-api/gram/cli/internal/mcp"
+	"github.com/speakeasy-api/gram/cli/internal/profile"
+	"github.com/speakeasy-api/gram/cli/internal/workflow"
+	"github.com/urfave/cli/v2"
+)
+
+func newInstallClaudeCodeCommand() *cli.Command {
+	return &cli.Command{
+		Name:  "claude-code",
+		Usage: "Install a Gram toolset as an MCP server in Claude Code",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:  "toolset",
+				Usage: "The slug of the Gram toolset to install (e.g., speakeasy-admin). Will automatically look up MCP configuration.",
+			},
+			&cli.StringFlag{
+				Name:  "toolset-url",
+				Usage: "The full MCP URL of the toolset (e.g., https://mcp.getgram.ai/org/project/environment). Use this for manual configuration.",
+			},
+			&cli.StringFlag{
+				Name:  "name",
+				Usage: "The name to use for this MCP server in Claude Code (defaults to toolset name or derived from URL)",
+			},
+			&cli.StringFlag{
+				Name:  "api-key",
+				Usage: "The API key to use for authentication (falls back to profile API key if not provided)",
+			},
+			&cli.StringFlag{
+				Name:  "header-name",
+				Usage: "The HTTP header name for the API key (defaults to Gram-Apikey)",
+				Value: "Gram-Apikey",
```

Comment:
Sending the header like this doesn't work for me and I'm not exactly sure as to why. I always send the header as `Authorization`, but I know there's a way to make another incantation work. I can just never seem to figure it out. Just wanna be absolutely sure that this works for you though